### PR TITLE
Added plausible_static exception in case of point to point IP

### PR DESCRIPTION
**Pull request by Liberasys**
At online.net, we have a specific IP/32 and a specific gateway. This is a point to point IP configuration.
In this case, the tests fails. I added a condition upon the /32 netmask. Hoping that administrator knowns what he does :-)

### CODE REVIEW

**Comment by sduthil** [comment]
You mean you have:

- a static IP address
- with netmask 255.255.255.255
- and a default gateway address that is not in any network known to the Wazo

Am I correct?
What happens without your patch?
Did you test your patch? Because there's a syntax error in it :)

File: xivo/xivo_config.py [hunk 1]
```diff
@@ -284,6 +284,11 @@ def plausible_static(static, schema):
     """
     address = network.parse_ipv4(static['address'])
     netmask = network.parse_ipv4(static['netmask'])
+
+    # If point to point configuration (netmask is 255.255.255.255), bypass the tests
+    if netmask[0] == 255 and netmask[1] == 255 and netmask[2] == 255 and netmask[3] == 255 and:
```
Comment (sduthil) [inline xivo/xivo_config.py]: The last `and:` causes a SyntaxError, doesn't it?

**Comment by Liberasys** [comment]
That's it.
Sorry, I left a problem while editing the patch on an other computer.
I think the good code is:
```
    # If point to point configuration (netmask is 255.255.255.255), bypass the tests
    if netmask[0] == 255 and netmask[1] == 255 and netmask[2] == 255 and netmask[3] == 255:
```
The error message is the one when plausible_static returns False.
Sorry, I had problems after changing HTTPS certificates, it disappointed me and I am testing ISSABEL on my VM at now, so I can't recover log message.

**Comment by pc-m** [comment]
I'm closing this PR since it did not get any development in a long time.

please reopen if this is still relevant to you.

**Pull request closed without merging**